packages/SwingSet/test/change-parameters/vat-carol.js

```diff
@@ -0,0 +1,9 @@
+import { Far } from '@endo/marshal';
+
+export function buildRootObject() {
+  return Far('root', {
+    doSomething() {
+      console.log('carol does something');
```

Comment:
maybe this could return the string, instead of adding bulk to our already-noisy test logs (in the successful case)

Reply:
Actually, the `console.log` is just there to give the method something to do as if it was a real method, and the method only exists to be sure the Carol vat is actually alive.  It was useful to see the message as I was getting the test working, but I can take the log message out now that the test works.  There's no reason to return a value -- nothing gets checked there.